src/components/Article.jsx

```diff
@@ -0,0 +1,16 @@
+import { string } from 'prop-types';
+import React from 'react';
+
+export const Article = ({ title, date, text }) => (
+  <>
+    <h1>{title}</h1>
+    <span>{date}</span>
+    <p>{text}</p>
+  </>
```

Comment:
You can use semantic tag <article>

Reply:
Arsen is right. Use `<article>` tag instead of a fragment